extensions/amp-story/0.1/icons.js

```diff
@@ -14,17 +14,18 @@
  * limitations under the License.
  */
 
- /* eslint-disable max-len */
+/* eslint-disable max-len */
+/** @private @const {!Object<string, string>} */
 export const ICONS = {
   mail:
-      `<svg xmlns="http://www.w3.org/2000/svg" width="32px" height="32px" viewBox="0 0 24 24">
-        <path d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 4l-8 5-8-5V6l8 5 8-5v2z"/>
-        <path d="M0 0h24v24H0z" fill="none"/>
-      </svg>`,
+      '<svg xmlns="http://www.w3.org/2000/svg" width="32px" height="32px" viewBox="0 0 24 24">' +
```

Comment:
SVGs can definitely be created in JS. But a different question I have here: our approach so far been that we put all icon SVGs in CSS. Why not do it here as well?